raiden/network/proxies/service_registry.py

```diff
@@ -75,6 +100,26 @@ def get_service_url(
             return None
         return result
 
+    def current_price(self, block_identifier: BlockSpecification) -> TokenAmount:
+        """Gets the currently required deposit amount."""
+        return self.proxy.contract.functions.currentPrice().call(block_identifier=block_identifier)
+
+    def token_address(self, block_identifier: BlockSpecification) -> AddressHex:
```

Comment:
Can you also return the bytes address here? We need to be consistent.

Reply:
OK!